### DISCUSSION TITLE
blocklist for description

### DISCUSSION THREAD

**Discussion by ewonais** [totaldebug/atomic-calendar-revive#1496] [Feature Requests]
Makes it easy  to block cancelled events that only has changed its description and not been removed. 
Currently there is an blocklist for event title and location but not for description.

**Comment by marksie1988** [comment]
Closing as a duplicate of the higher-voted #1165 ("Filter on Event Description"), which covers the same use case. Please add your 👍 there so the canonical request reflects the demand.
